### ISSUE TITLE
Feature Request - Additional Cinematic aspect Ratio

### ISSUE THREAD

**Issue by lord-lethris** [RockOfFire/ComfyUI_Comfyroll_CustomNodes#24]
Sorry, I couldn't find A Feature request or discussion area.

Is it possible to add the 1.85:1 widescreen and an anamorphic 2.39:1 widescreen Cinematic aspect Ratios to both SD and SDXL CR Aspect Ratio Nodes?

Thanks In Advance

Is it also possible to combine the CR Aspect Ratio Nodes into One node, maybe with a SD/SDXL switch.  Reason: Because I know quite a few of us who switch between SD and SDXL within the same workflow - saves cluttering up multiple nodes that essentially do the same thing.

**Comment by RockOfFire** [comment]
We can add the Aspect Ratios.  :D  For the 2nd thing, that would require doing some Javascript.  It's something we could do in the future though.

**Comment by Suzie1** [comment]
Hi again,
I'm adding 1.85:1 and 2.39:1 on SD1.5. This will go out in today's update.
For SDXL it is less clear what ratios to use, since they need to be divisible by 64.
I currently have the 21:9 landscape 1536x640 on SDXL which is close to anamorphic but not quite.
https://www.reddit.com/r/StableDiffusion/comments/15c3rf6/sdxl_resolution_cheat_sheet/
What I could do is add a new node for precise cinema format cropping. This could crop output to exact cinema sizes or add black bars top and bottom. What do you think?
Suzie

**Comment by lord-lethris** [comment]
Whatever you think is best - black bars top and bottom might be the way to go, as you would obtain the full anamorphic render.

Thanks for doing this BTW 💞

**Comment by Trobinou** [comment]
Hello,
I also agree that there should only be one node for choosing the aspect ratio. I currently use the following solution, but it's not very sexy:

![Aspect-ratio](https://github.com/RockOfFire/ComfyUI_Comfyroll_CustomNodes/assets/132901709/a769fce1-accd-45e0-a737-eade16f47d5f)

If you decide to merge the 2 nodes, would it also be possible to have a 'Latent' output directly?

Thanks in advance 😊

**Comment by Suzie1** [comment]
Yes, I have been thinking about including the latent output. I can add this
node in the next update.

**Comment by lord-lethris** [comment]
> Hello, I also agree that there should only be one node for choosing the aspect ratio. I currently use the following solution, but it's not very sexy:
> 
> ![Aspect-ratio](https://user-images.githubusercontent.com/132901709/283998382-a769fce1-accd-45e0-a737-eade16f47d5f.png)
> 
> If you decide to merge the 2 nodes, would it also be possible to have a 'Latent' output directly?
> 
> Thanks in advance 😊

that's exactly what I have setup :D - I have a switch that selects between the two -it also switches the upscailer from 512 to 1024 and various other things

**Comment by Trobinou** [comment]
@Suzie1 
Many thanks!

**Comment by Suzie1** [comment]
Hi, a new node **CR Aspect Ratio** is being added in the update later today. This combines the sizes for both SDXL and SD1.5.
I have also added an empty latent output on all the aspect ratio nodes. Suzie

**Comment by Trobinou** [comment]
Great news! I'm really looking forward to it 😊

**Comment by Trobinou** [comment]
Hello @Suzie1 

Thank you for these changes, which greatly simplify the aesthetics of the workflow 👍
If I may say so, there's still one point that could be improved: once you've chosen the aspect ratio, you can't tell at a glance whether the selected format is SD1.5 or SDXL, unless you pay attention to the dimensions and know them perfectly well.
Would it be possible to add 'SD1.5' and 'SDXL' in front of the resolutions?
example: `SD1.5 - 1:1 square 512×512`

**Comment by Suzie1** [comment]
thats a good idea, i'll get rid of the sub-headings and use the SD1.5 and SDXL prefixes

**Comment by Trobinou** [comment]
Many thanks, it's perfect! 💯👏👍

**Comment by Suzie1** [comment]
Closing, thanks for your input :)

**Comment by lord-lethris** [comment]
thanks for the heads up, really appreciate it 💞- i'll check now (good job you told me as one of my workflows is up for a competition 😱)

**Comment by lord-lethris** [comment]
@Suzie1 Oh no - Empty Latent Image Isn't working 😱😱

![image](https://github.com/Suzie1/ComfyUI_Comfyroll_CustomNodes/assets/1736720/d8f7c4ba-9b2c-4e06-ac1d-acc27dab5a9f)

**Comment by Suzie1** [comment]
Hi, you need to remove and reload the node.

**Comment by lord-lethris** [comment]
@Suzie1 - I did :(

**Comment by Suzie1** [comment]
Sorry, I'm in bed. I'll retest in the morning.

**Comment by lord-lethris** [comment]
I have put a Temporary fix in place for now so no rush..
"We have a work around"

**Comment by Suzie1** [comment]
Hello, I tested the live version that is currently on github and it works ok.

Normally when a node has a workflow impacting change, then its best to update the node pack, then restart comfy, delete the node in the workflow and then replace it. Looking at your image above, I can see show_help twice which suggests the node needs deleting and replacing.

We did a repository transfer yesterday and this could have affected your update. Until that transfer is complete in ComfyUI manager is may be best to do an update using git pull or downloading the zip file (remove -main from the folder name).
If this is still giving problems could you DM me on AI Revolution or ComfyUI Workflow Dev.

Suzie

**Comment by lord-lethris** [comment]
cool :) - I fixed it by removing the node and re-installing it again.

**Comment by Suzie1** [comment]
i'm glad it's working again, sorry for the problems
